packages/repository/src/__tests__/unit/mixins/add-method-to-repository.mixin.unit.ts

```diff
@@ -0,0 +1,52 @@
+// Copyright IBM Corp. 2020. All Rights Reserved.
+// Node module: @loopback/repository
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {expect} from '@loopback/testlab';
+import {DataSource} from 'loopback-datasource-juggler';
+import {Note} from '../../fixtures/models/note.model';
+import {NoteRepository} from '../../fixtures/repositories/note.repository';
+
+describe('add method to crud repository via mixin', () => {
+  let ds: DataSource;
+  let repo: NoteRepository;
+  const noteData = {
+    title: 'groceries',
+    content: 'eggs,bacon',
+    category: 'keto',
+  };
+  let note: Note;
+
+  beforeEach(async () => {
+    ds = new DataSource({
+      connector: 'memory',
+    });
+    repo = new NoteRepository(ds);
+  });
+
+  // method from CrudRepository
+  it(`non-mixin method 'create' exists`, async () => {
+    expect(repo.create).to.be.a.Function();
```

Comment:
In fact, calling `repo.create` on the next line is already covering the assertion `expect(repo.create).to.be.a.Function()`. If `create` is not a method (a function), then the JavaScript runtime will throw an error that will fail the test:

```
Uncaught TypeError: repo.create is not a function
```

IMO, the error message reported by the runtime will be actually more helpful than the assertion message, because it can describe both the variable (`repo`) and the member name (`create`). The assertion message will just say something like `expected undefined to be a function`.

I am proposing to remove these `.to.be.a.Function()` assertions from all three tests in this file.